packages/web-components/src/components/logo-grid/logo-grid.scss

```
@@ -0,0 +1,10 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '@carbon/ibmdotcom-styles/scss/components/logo-grid/logo-grid';
```

Doing below may eliminate the Sass compilation error we see in the CI:

```suggestion
@import 'carbon-components/scss/globals/scss/vars';
@import '../../globals/scss/vars';
@import '../../globals/scss/box-sizing';
@import '@carbon/ibmdotcom-styles/scss/components/logo-grid/logo-grid';
```